pkg/ui/workspaces/cluster-ui/src/jobs/jobDetailsPage/jobDetails.tsx

```diff
@@ -165,19 +165,6 @@ export class JobDetails extends React.Component<
                 <JobStatusCell job={job} lineWidth={1.5} hideDuration={true} />
               }
             />
-            {hasNextRun && (
```

Comment:
The `nextRun` and `hasNextRun` parameters can be removed as well as their corresponding input arguments.

Reply:
Done!